src/test/cs/Infinispan/HotRod/RemoteContQueryTestSuite.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Infinispan.HotRod.Tests;
+using Infinispan.HotRod.Tests.Util;
+using NUnit.Framework;
+using System.Collections;
+
+namespace Infinispan.HotRod.TestSuites
+{
+    public class RemoteContQueryTestSuite
+    {
+        HotRodServer server;
+
+        [TestFixtureSetUp]
+        public void BeforeSuite()
+        {
+            server = new HotRodServer("clustered-indexing.xml");
```

Comment:
We already have the RemoteQueryTestSuite which runs against clustered-indexing.xml. Maybe we could add the ContinuousQueryTest to the RemoteQueryTestSuite and save one start of the server with each test run?